modules/spark/src/main/scala_2.11/spark-last/Boot.scala

```diff
@@ -6,12 +6,12 @@ import scala.tools.nsc.Settings
 import java.io.File
 
 object Boot {
-  def classServer(outputDir:File) = {
+  /*def classServer(outputDir:File) = {
     val conf = new SparkConf()
     val server = new HttpServer(conf, outputDir, new SecurityManager(conf))
     server
-  }
-  
+  }*/
```

Comment:
why do you comment this out?

maybe delete or at least add a `FIXME` comment, if it needs attention in the future.